lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriterOptions.java

```diff
@@ -48,10 +51,16 @@ public int getMaxPageSize()
         return maxPageSize;
     }
 
+    public int getRowGroupMaxRowCount()
+    {
+        return rowGroupMaxRowCount;
```

Comment:
starting it with `maxRowGroup...` makes it closer to existing `maxRowGroupSize`.
`maxRowGroupCardinality`?

Also place the new property next to the existing one.

Reply:
I based the name off of a similar property for ORC, but I don't have much of a preference https://github.com/trinodb/trino/blob/master/lib/trino-orc/src/main/java/io/trino/orc/OrcWriterOptions.java#L63

Reply:
you convinced me.